Snakefile

```diff
@@ -193,42 +157,23 @@ rule reconstruct:
 rule parse_output:
     input: raw_file = os.path.join(out_dir, 'raw-pathway-{dataset}-{algorithm}-{params}.txt')
     output: standardized_file = os.path.join(out_dir, 'pathway-{dataset}-{algorithm}-{params}.txt')
-    # run the post-processing script
     run:
         PRRunner.parse_output(wildcards.algorithm, input.raw_file, output.standardized_file)
 
 # Write the mapping from parameter indices to parameter dictionaries
-# TODO: Need this to have input files so it updates
```

Comment:
We still need this TODO and the next line.  We don't have a fully functional solution to making sure the log files also match what is in the config file.